great_expectations/cli/datasource.py

```diff
@@ -428,6 +429,10 @@ def _add_sqlalchemy_datasource(context, prompt_for_datasource_name=True):
             if not load_library("snowflake", install_instructions_string="pip install snowflake-sqlalchemy"):
                 return None
             credentials = _collect_snowflake_credentials(default_credentials=credentials)
+        elif selected_database == SupportedDatabases.BIGQUERY:
+            if not load_library("pybigquery", install_instructions_string="pip install pybigquery"):
+                return None
+            credentials = _collect_bigquery_credentials(default_credentials=credentials)
```

Comment:
Eventually we'll want integration tests that cover this. But that is not worth blocking this great PR!

Reply:
Experimental!